README.md

```diff
@@ -58,6 +58,15 @@ installers for all supported operating systems.
     sudo apt-get install etcher-electron
     ```
 
+##### Uninstall
+
+```sh
+sudo apt-get remove etcher-electron
+sudo apt-key del D401AB61
```

Comment:
I agree with @lurch, and suggest removing the key removal step, but keeping the uninstall instructions. Can't hurt to have them – at least people looking for it will find it quickly, without having to search the interwebs.